Sources/App/Controllers/PackageController+routes.swift

```diff
@@ -152,11 +152,18 @@ struct PackageController {
                     documentationVersions.filter { $0.latest == .preRelease }.first
```

Comment:
Unrelated but I just noticed it:

```suggestion
                    documentationVersions.first { $0.latest == .preRelease }
```

(also the line above but I can select that as part of the diff.)